jansi/src/test/java/org/fusesource/jansi/FilterPrintStreamTest.java

```diff
@@ -0,0 +1,30 @@
+package org.fusesource.jansi;
+
+import java.io.OutputStream;
+import java.io.PrintStream;
+
+import org.junit.Test;
+
+public class FilterPrintStreamTest
+{
+    @Test
+    public void testPrintMethods()
+        throws Exception
+    {
+        PrintStream ps = new FilterPrintStream(System.out);
+        ps.println( "String" );
+        ps.println( '€' );
+        ps.close();
+    }
+
+    @Test
+    public void testWrite()
+        throws Exception
+    {
+        OutputStream os = new FilterPrintStream(System.out);
+        os.write( 'A' );
+        os.write( 'B' );
+        os.write( '€' );
```

Comment:
This is actually an illegal method call.
Even if the parameter is an integer,  the assumption is that it's a byte, so between 0 and 255.

Maybe using `"€".getBytes()` ? Which I've done and hit the missing method above.